.github/workflows/go.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.20
```

Comment:
Please convert it to `go-version-file` (https://github.com/nspcc-dev/neofs-node/blob/9756cc40e2fe2cf7f7dfb2e56021342f794db1e5/.github/workflows/go.yml#L47), it's so nice I want it everywhere. It will require doing checkout first though.

Reply:
Great advise, thanks!